frontend/src/App.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+
+import Header from "./Components/Headers/Header";
+import Products from "./Components/Products/Products";
+import Items from "./Components/Items/Items";
+
+import styles from "./App.module.scss";
+
+const App = () => {
+  const [linkToken, setLinkToken] = useState<string | null>("");
+  const [linkSuccess, setLinkSuccess] = useState(false);
+  const [isItemAccess, setIsItemAccess] = useState(false);
```

Comment:
can you help me understand what `isItemAccess` is doing?

Reply:
isItemAccess sets whether or not we have received an access_token and item_id, and displays error message on page if not.

Reply:
Because they determine whether or not products and items are rendered in the app component, i needed a boolean that is sent back to the app component when access_token and item_id are set later in the link token